packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -215,9 +215,9 @@ class ScrollPosition extends ViewportOffset {
   /// scroll notifications will be dispatched. No overscroll notifications can
   /// be generated by this method.
   ///
-  /// Immediately after the jump, a ballistic activity is started, in case the
-  /// value was out of range.
-  void jumpTo(double value) {
+  /// If settle is true then, immediately after the jump, a ballistic activity
+  /// is started, in case the value was out of range.
+  void jumpTo(double value, { bool settle: true }) {
```

Comment:
Ok. Please add a clear TODO here saying that this API is temporary and describe it as such in the dartdocs. Maybe even use a different method name and mark that method @deprecated, with an `//ignore` where you use it?

The point is that this is clear technical debt, and we should label it as such.